pixi.js/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Pixi.js v4.3.0
-// Project: https://github.com/pixijs/pixi.js/
-// Definitions by: Clark Stevenson <https://github.com/clark-stevenson>
+// Type definitions for Pixi.js 4.3.4
```

Comment:
Can you keep this at 4.3? We don't keep anything other than major & minor version numbers.